src/js/main.js

```diff
@@ -366,6 +368,61 @@ Site.Earth = {
   init: function() {
     var _this = this;
 
+    if (_this.canHandleThree()) {
+      _this.initThreeEarth();
+    } else {
+      _this.initFlatEarth();
+    }
+
+  },
+
+  canHandleThree: function() {
+    var _this = this;
+
+    // init MobileDetect
+    var md = new MobileDetect(window.navigator.userAgent);
+
+    if (md.is('iOS') || md.is('Safari')) {
```

Comment:
this also needs to test for desktop safari

Reply:
Can we recheck this?

It successfully tells between desktop chrome and desktop safari, for me.